src/InfoViz/Native/MutualInformationDiagram/index.js

```diff
@@ -187,7 +187,7 @@ function informationDiagram(publicAPI, model) {
 
     if (variableList.length < 2 || !model.container) {
       // Select the main circle and hide it and unhide placeholder
-      d3.select(model.container).select('svg.information-diagram').classed(style.hidden, true);
+      d3.select(model.container).select('svg.information-diagram').attr('class', style.informationDiagramSvgHide);
```

Comment:
> ...It sets the entire 'class' attribute, rather than adding or subtracting from it. If I use .classed, I have to set style.informationDiagramSvgShow to false, then style.informationDiagramSvgHide to true. So it's less verbose. Is it too unclear?

I would probably not realize what was going on. But that doesn't mean it's bad practice.